include/roboteam_ai/new_skills/NewSkill.h

```diff
@@ -0,0 +1,77 @@
+//
+// Created by jessevw on 21.02.20.
+//
+
+#ifndef RTT_NEWSKILL_H
+#define RTT_NEWSKILL_H
+
+#include <include/roboteam_ai/treeinterp/Leaf.h>
+
+#ifndef ROBOTEAM_AI_SKILL_H
+#define ROBOTEAM_AI_SKILL_H
+
+#include <roboteam_utils/Angle.h>
+#include "include/roboteam_ai/utilities/IOManager.h"
```

Comment:
Include naming. Also, why do you have two header guards?

Reply:
Whoops I copied it from the old skill and forgot to change it. I basically removed the things I thought we didnt need as quite a lot of functionality can be kept from the old implementation